gef.py

```diff
@@ -1847,28 +1848,28 @@ def gdb_get_nth_next_instruction_address(addr, n):
     return insn.address
 
 
-def gef_instruction_n(addr, n):
+def gef_instruction_n(addr: int, n: int):  # -> Instruction
```

Comment:
Why the comments?

Reply:
with python 3.6 type hints can only be handled in the correct order. for this to work without throwing an error in py3.6 the order of the code would have to be modified. I thought about leaving it here for a later version but I can also remove the comments (I anyway wasn't 100% consequent across the codebasis)

Reply:
Ah. So we can move  `class Instruction` higher up to fix? This is fine as is, but let's file a ticket to have that done so we don't forget in the future.

Reply:
or we wait until the right python version. iirc there are some "circular type hints" that are not possible to solve by changing the order. and also e.g. in class `Permission` with py3.6 we can't reference the `Permission` type itself...

Reply:
sure